.eslintignore

```diff
@@ -1,2 +1,3 @@
 coverage
 lib
+pnpm-lock.yaml
```

Comment:
[Request] Adding `pnpm-lock.yaml` is a good start. But without `!.*` added the `.github/*/*.yml` files aren't getting linted!

```suggestion
!.*
coverage
lib
node_modules
pnpm-lock.yaml
```

In my local test it looks like `.release-it.json` and `.vscode/launch.json` have some discovered out-of-order keys. It's fine to include those in this PR too. Splitting up PRs gets annoying after a while heh.

Reply:
Done, it does indeed :smile:

Reply:
Yes please to the new issue, that'd be great!